### PR TITLE
Added Docs page for Postmark

### PR DESCRIPTION
**Pull request by mohanthilak**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added docs for Postmark using the SendGrid example.
- **Why was this change needed?** (You can also link to an open issue here)
The intent is to provide helpful docs for users to use novu's platform easily.
- **Other information**:
Created by seeing SendGrid as an example.  Willing to take any feedback and make the required changes.
Ref: #1349

### CODE REVIEW

File: docs/docs/channels/email/postmark.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Before you send email in full scale, you will need to authenticate your sender's indentity. This is due to the latest regulatory changes regarding SPAM rules and email fraud. Most of the providers including Postmark require you to authenticate your sender identity before you can send emails.
+
+
+Postmark allows you to authenticate your sender identity using one of the following methods:
+
+- [Single Sender Verification](https://account.postmarkapp.com/signatures/new) - This is the easiest way to authenticate your sender identity.
+- [Entire Domain Authentication](https://postmarkapp.com/support/article/1046-how-do-i-verify-a-domain#:~:text=be%20verified%20automatically.-,Navigate%20to%20Sender%20Signatures.,to%20your%20DNS%2C%20choose%20Verify.) - This is recommended if you are sending emails from multiple accounts under your domain.
+
+## Create a SendGrid integration with Novu
```
Comment (p-fernandez) [inline docs/docs/channels/email/postmark.md:24]:
```suggestion
## Create a Postmark integration with Novu
```

**Review by p-fernandez — CHANGES_REQUESTED** [review]
🌟 
Just a small change and we are good to go. Thanks!

File: docs/docs/channels/email/postmark.md (hunk 1 above)
Comment (mohanthilak) [reply docs/docs/channels/email/postmark.md]: Oh, my bad. I overlooked that. Thanks @p-fernandez

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**